fixtures/u2_func_simple.rb.sem.uast

```diff
@@ -0,0 +1,166 @@
+                              start: { '@type': "uast:Position",
+                                 offset: 23,
+                                 line: 2,
+                                 col: 5,
+                              },
+                              end: { '@type': "uast:Position",
+                                 offset: 27,
+                                 line: 2,
+                                 col: 9,
+                              },
+                           },
+                           value: { '@type': "int",
+                              '@token': 1,
+                              '@role': [Binary, Expression, Literal, Number, Primitive, Right],
+                              '@pos': { '@type': "uast:Positions",
+                                 start: { '@type': "uast:Position",
+                                    offset: 27,
+                                    line: 2,
+                                    col: 9,
+                                 },
+                                 end: { '@type': "uast:Position",
+                                    offset: 27,
+                                    line: 2,
+                                    col: 9,
+                                 },
+                              },
+                           },
+                        },
+                        { '@type': "return",
+                           '@role': [Return, Statement],
+                           '@pos': { '@type': "uast:Positions",
+                              start: { '@type': "uast:Position",
+                                 offset: 33,
+                                 line: 3,
+                                 col: 5,
+                              },
+                              end: { '@type': "uast:Position",
+                                 offset: 38,
+                                 line: 3,
+                                 col: 10,
+                              },
+                           },
+                           target: { '@type': "uast:Identifier",
+                              '@role': [Binary, Left],
+                              '@pos': { '@type': "uast:Positions",
+                                 start: { '@type': "uast:Position",
+                                    offset: 40,
+                                    line: 3,
+                                    col: 12,
+                                 },
+                                 end: { '@type': "uast:Position",
+                                    offset: 40,
+                                    line: 3,
+                                    col: 12,
+                                 },
+                              },
+                              Name: "a",
+                           },
+                        },
+                     ],
+                  },
+               ],
+               Type: { '@type': "uast:FunctionType",
+                  Arguments: { '@type': "args",
```

Comment:
`Arguments` should be an array of `uast:Argument`s

Reply:
fixed in next commit